### ISSUE TITLE
nimsuggest cannot find identifiers generated futhark

### ISSUE THREAD

**Issue by jackhftang** [PMunch/futhark#62]
I am able to compile, link and run the code. The only problem is that nimsuggest cannot lookup any identifiers. I dig a bit and think that futhark should have generated declaration like this `proc tsparsernew(): ptr Tsparser_436207981 {.cdecl, importc: "ts_parser_new".}`. It seems to me it is problem of nimsuggest. I want to confirm that are you experiencing the same problem or it is just me? 

![Screenshot from 2023-03-17 13-23-55](https://user-images.githubusercontent.com/1190545/225819755-ca4b98ff-1464-4fa1-a755-41887065c5eb.png)

**Comment by PMunch** [comment]
Yes, this is a known limitation with Nimsuggest. Essentially it is just Nimsuggest which doesn't properly run the macro. If you follow the suggestion in the ["Shipping Wrappers"](https://github.com/PMunch/futhark#shipping-wrappers) section of the README it will work. This is because Nimsuggest is able to take then non-futhark path and just grab the cached file instead.
